q2_diversity/_ordination.py

```diff
@@ -34,19 +34,25 @@ def pcoa_biplot(pcoa: skbio.OrdinationResults,
 
 
 def tsne(distance_matrix: skbio.DistanceMatrix,
-         number_of_dimensions: int = 3) -> skbio.OrdinationResults:
+         number_of_dimensions: int = 3,
```

Comment:
I think we should set the default value to 2, since this is the more common use of t-SNE - although PCoA reduces to three dimensions by default